### PR TITLE
docs/selenium_manager: mention about Python now supporting ENV for driver path

### PR DESCRIPTION
**Pull request by Delta456**
### **User description**
…iver path

**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

### Description

Mentions Python support for setting ENV for driver path

### Motivation and Context

New implemented feature which needs to be documented

image

### Types of changes

- [x] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

___

### **PR Type**
Documentation

___

### **Description**
- Added documentation for Python support for setting the driver path using environment variables.
- Updated the documentation to reflect that this feature is available in the Selenium Python binding starting from version 4.26.0.
- Improved formatting by enclosing environment variable names in backticks for clarity.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>selenium_manager.en.md</strong><dd><code>Document Python support for environment variable driver path</code></dd></summary>
<hr>

website_and_docs/content/documentation/selenium_manager.en.md

<li>Added Python to the list of bindings supporting environment variable <br>for driver path.<br> <li> Updated version information for Python binding support.<br> <li> Formatted environment variable names with backticks.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/2055/files#diff-abfd3399fa93ee23ed8a6b8637b566cf12c67d477fda705ed877a10f013047b7">+7/-6</a>&nbsp; &nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>selenium_manager.ja.md</strong><dd><code>Document Python support for environment variable driver path</code></dd></summary>
<hr>

website_and_docs/content/documentation/selenium_manager.ja.md

<li>Added Python to the list of bindings supporting environment variable <br>for driver path.<br> <li> Updated version information for Python binding support.<br> <li> Formatted environment variable names with backticks.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/2055/files#diff-a84369b32708be4deb5da8f23f5b2c0adecb18e5eee5d30601e96f5d75e999f0">+7/-6</a>&nbsp; &nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>selenium_manager.pt-br.md</strong><dd><code>Document Python support for environment variable driver path</code></dd></summary>
<hr>

website_and_docs/content/documentation/selenium_manager.pt-br.md

<li>Added Python to the list of bindings supporting environment variable <br>for driver path.<br> <li> Updated version information for Python binding support.<br> <li> Formatted environment variable names with backticks.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/2055/files#diff-d8c0fd932e504044c81c242c7e255cb4533a7ea66e2b12d142c1cbb1680f6213">+7/-6</a>&nbsp; &nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>selenium_manager.zh-cn.md</strong><dd><code>Document Python support for environment variable driver path</code></dd></summary>
<hr>

website_and_docs/content/documentation/selenium_manager.zh-cn.md

<li>Added Python to the list of bindings supporting environment variable <br>for driver path.<br> <li> Updated version information for Python binding support.<br> <li> Formatted environment variable names with backticks.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/2055/files#diff-7b85ea66160081776015671c1e38dda6f8fd556f1217683dff9a0982fa71fc2b">+7/-6</a>&nbsp; &nbsp; &nbsp; </td>

</tr>
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

File: website_and_docs/content/documentation/selenium_manager.pt-br.md [hunk 1]
```diff
@@ -337,20 +337,21 @@ sudo apt-get install libatk-bridge2.0-0
...
 For example, to specify the path to the chromedriver,
 you can set the `SE_CHROMEDRIVER` environment variable to the path of the chromedriver executable.
 The following bindings allow you to specify the driver path using an environment variable:
 
 * Ruby
 * Java
+* Python
 
-This feature is available in the Selenium Ruby binding starting from version 4.25.0.
+This feature is available in the Selenium Ruby and Python binding starting from version 4.25.0 and 4.26.0.
```
Comment (harsha509) [inline website_and_docs/content/documentation/selenium_manager.pt-br.md]: This feature is available in the Selenium Ruby binding starting from version 4.25.0 and in the Python binding from version 4.26.0.

File: website_and_docs/content/documentation/selenium_manager.zh-cn.md [hunk 2]
```diff
@@ -337,20 +337,21 @@ sudo apt-get install libatk-bridge2.0-0
...
 For example, to specify the path to the chromedriver,
 you can set the `SE_CHROMEDRIVER` environment variable to the path of the chromedriver executable.
 The following bindings allow you to specify the driver path using an environment variable:
 
 * Ruby
 * Java
+* Python
 
-This feature is available in the Selenium Ruby binding starting from version 4.25.0.
+This feature is available in the Selenium Ruby and Python binding starting from version 4.25.0 and 4.26.0.
```
Comment (harsha509) [inline website_and_docs/content/documentation/selenium_manager.zh-cn.md]: same here

File: website_and_docs/content/documentation/selenium_manager.ja.md [hunk 3]
```diff
@@ -337,20 +337,21 @@ sudo apt-get install libatk-bridge2.0-0
...
 For example, to specify the path to the chromedriver,
 you can set the `SE_CHROMEDRIVER` environment variable to the path of the chromedriver executable.
 The following bindings allow you to specify the driver path using an environment variable:
 
 * Ruby
 * Java
+* Python
 
-This feature is available in the Selenium Ruby binding starting from version 4.25.0.
+This feature is available in the Selenium Ruby and Python binding starting from version 4.25.0 and 4.26.0.
```
Comment (harsha509) [inline website_and_docs/content/documentation/selenium_manager.ja.md]: This feature is available in the Selenium Ruby binding starting from version 4.25.0 and in the Python binding from version 4.26.0.

File: website_and_docs/content/documentation/selenium_manager.en.md [hunk 4]
```diff
@@ -337,20 +337,21 @@ sudo apt-get install libatk-bridge2.0-0
...
 For example, to specify the path to the chromedriver, 
 you can set the `SE_CHROMEDRIVER` environment variable to the path of the chromedriver executable.
 The following bindings allow you to specify the driver path using an environment variable:
 
 * Ruby
 * Java
+* Python
 
-This feature is available in the Selenium Ruby binding starting from version 4.25.0.
+This feature is available in the Selenium Ruby and Python binding starting from version 4.25.0 and 4.26.0.
```
Comment (harsha509) [inline website_and_docs/content/documentation/selenium_manager.en.md]: This feature is available in the Selenium Ruby binding starting from version 4.25.0 and in the Python binding from version 4.26.0.

**Review by harsha509 — APPROVED** [review]
Thank you @Delta456 !

**Review by harsha509 — CHANGES_REQUESTED** [review]
HI @Delta456 ,

Thank you for the PR. Requested few changes!

File: website_and_docs/content/documentation/selenium_manager.pt-br.md (hunk 1 above)
Comment (Delta456) [reply website_and_docs/content/documentation/selenium_manager.pt-br.md]: Sounds good!

File: website_and_docs/content/documentation/selenium_manager.zh-cn.md (hunk 2 above)
Comment (Delta456) [reply website_and_docs/content/documentation/selenium_manager.zh-cn.md]: Sure

**Review by harsha509 — APPROVED** [review]
Thank you @Delta456 !

**Pull request merged**